frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,63 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'normal' | 'stretch'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
+  parts: ['outerbox', 'innerbox'],
+  baseStyle: {
+    outerBox: {
+      padding: '4px',
+      borderRadius: '8px',
+      width: '177px',
+    },
+    innerBox: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      padding: '8px 16px',
+      height: '44px',
+      background: 'white',
+      boxSizing: 'border-box',
+      borderStyle: 'solid',
+      borderWidth: '1px',
+      borderRadius: '4px',
+      borderColor: 'white',
```

Comment:
not sure about this but i think inlining it into `outerBox` might be clearer!

Reply:
I kept the current implementation because I am using the outerBox padding background to provide the curved effect